### PR TITLE
Adding DeccanRubyConf

### PR DESCRIPTION
**Pull request by rishiip**
Adding Deccan RubyConf 2017 to the list of conferences. 😊

### CODE REVIEW

File: _data/current.yml [hunk 1]
```diff
@@ -68,6 +68,15 @@
   cfp_phrase: CFP closes
   cfp_date: "March 31, 2017"
 
+- name: Deccan RubyConf
+  location: Pune, India
+  dates: "Aug 12, 2017"
+  url: http://www.deccanrubyconf.org/
+  twitter: deccanrubyconf
+  reg_phrase: Registration is open
+  cfp_phrase: CFP open
```
Comment (jonallured) [inline _data/current.yml]: I think this should be "CFP closes" so that the line makes sense.

**Comment by rishiip** [comment]
@jonallured done :blush:

**Comment by jonallured** [comment]
Thanks for the PR - I just deployed the site!! :D

**Comment by rishiip** [comment]
👍

**Pull request closed without merging**